scripts/convert-docs.js

```diff
@@ -22,6 +29,20 @@ const convertDocs = (docs) => {
     fs.writeFile(fileName, content, (err) => {
       if (err) logger.error(`Could not create ${fileName}.`);
     });
+
+    if (process.env.DEBUG) {
```

Comment:
is this turning it back into html?

Reply:
ooops nevermind I read the description! Ignore this comment :)

Reply:
Ish. It is writing the raw HTML back to a file with a `.html` extension. I used this a couple times and thought it might be helpful to compare the output we get from turndown with the original HTML.